example/docs/.vuepress/components/home-page.vue

```diff
@@ -192,27 +192,27 @@
               <b>Cropper</b>
             </p>
             <p>
-              is the root component of this library. It contains coordinates of current cropped area (left, top, width, height) relative to original image coordinates. This coordinates can be imagined like <b>box</b>.
-              It responsible for:
+              is the root component of this library. It contains the dimensions of the currently cropped area (left, top, width, height) relative to the original image dimensions. These dimensions can be imagined like a <b>box</b>.
```

Comment:
I assume that dimensions means only size, but the coordinates might include position also, so it's the more suitable word. Am I wrong?

Reply:
You're right, I understand now. Thanks for the clarification!